test/blackbox-tests/test-cases/pkg/ocaml-compiler.t

```diff
@@ -0,0 +1,110 @@
+Setting the compiler in the lock directory
+
+We need some data for ocamlc -config
+
+  $ mkdir stdlib && touch stdlib/Makefile.config
+  $ . ./helpers.sh
+  $ cat >ocaml.config <<EOF
```

Comment:
This could be inlined into `mycamlsources/ocamlc`, I assume?

Reply:
It could be, but it needs to be used in two different places:

* The fake OCaml used to shadow the system OCaml for reproducibility
* The fake OCaml installed from the lock dir

So if it's a file, we can reuse it in both instances.